Collaboration2/Collaboration2/Scenes/Search Page/SearchPageViewModel.swift

```diff
@@ -48,7 +78,8 @@ class SearchPageViewModel: ObservableObject {
             .store(in: &cancellables)
     }
     
-    init() {
+    init(modelContext: ModelContext) {
```

Comment:
I think initialization has to be in lifecycles, above functions and under properties. maybe on 20 th line